nextflow.config

```diff
@@ -92,86 +100,76 @@ includeConfig 'conf/base.config'
 // Load modules.config for DSL2 module specific options
 includeConfig 'conf/modules.config'
 
-// Load ref_databases.config for all pipelines
+// Load ref_databases.config for reference taxonomy
 includeConfig 'conf/ref_databases.config'
- 
+
 // Load nf-core custom profiles from different Institutions
 try {
     includeConfig "${params.custom_config_base}/nfcore_custom.config"
 } catch (Exception e) {
     System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
 }
-// Load nf-core/ampliseq custom profiles from different Institutions
-try {
-    includeConfig "${params.custom_config_base}/pipeline/ampliseq.config"
-} catch (Exception e) {
-    System.err.println("WARNING: Could not load nf-core/config/ampliseq profiles: ${params.custom_config_base}/pipeline/ampliseq.config")
-}
 
 profiles {
+    debug { process.beforeScript = 'echo $HOSTNAME' }
     conda {
-        docker.enabled = false
-        singularity.enabled = false
-        podman.enabled = false
-        shifter.enabled = false
-        charliecloud.enabled = false
+        params.enable_conda    = true
+        docker.enabled         = false
+        singularity.enabled    = false
+        podman.enabled         = false
+        shifter.enabled        = false
+        charliecloud.enabled   = false
     }
-    debug { process.beforeScript = 'echo $HOSTNAME' }
-    conda { params.enable_conda = true }
     docker {
-        docker.enabled = true
-        singularity.enabled = false
-        podman.enabled = false
-        shifter.enabled = false
-        charliecloud.enabled = false
-        // Avoid this error:
-        //   WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
-        // Testing this in nf-core after discussion here https://github.com/nf-core/tools/pull/351
-        // once this is established and works well, nextflow might implement this behavior as new default.
-        docker.runOptions = '-u \$(id -u):\$(id -g)'
+        docker.enabled         = true
+        docker.userEmulation   = true
```

Comment:
```suggestion
        docker.runOptions = '-u \$(id -u):\$(id -g)'
```

Reply:
I know it is in the template, but Paolo said that was not a good option, it will probably be changed again in the new release

Reply:
Thanks!